docs/gettingstarted.md

```diff
@@ -0,0 +1,137 @@
+# Getting started
+
+---
+
+## Table of Contents
+
+ - [Installation and environment set up](#installation-and-environment-set-up)
+ - [Jupyter Notebooks](#jupyter-notebooks)
+ - [First model](#first-model)
+ - [Elements of the model](#elements-of-the-model)
+
+---
+
+
+## Installation and environment set up
+
+Please note that for development, we suggest using [Poetry](https://python-poetry.org/) for dependency management and packaging. Instructions for setting up the Python environment with Poetry are in the section [Reference](/reference/#set-up-environment-with-poetry).
+
+### 0. Pre-installation Virtual Environments with [`venv`](https://docs.python.org/3/library/venv.html) (Optional):
+If you wish to create an easy to use virtual environment to install radCAD inside of, please use the built in `venv` package [see: [documentation about venv](https://docs.python.org/3/library/venv.html)].
+
+**Create** a virtual environment:
+```bash
+$ python3 -m venv venv
+```
+
+**Activate** an existing virtual environment (Unix/macOS):
+```bash
+$ source venv/bin/activate
+```
+
+**Activate** an existing virtual environment (Windows):
+```bash
+$ source venv\Scripts\activate
+```
+
+### 1. Installation: 
+Requires [>= Python 3.8](https://www.python.org/downloads/) 
+
+Install Using [pip](https://pypi.org/project/cadCAD/)
+```bash
+$ pip install radCAD
+```
+
+### 2. Install Python dependencies inside virtual environment
+
+```bash
+pip install -r requirements.txt
+```
+
+
+
+## Jupyter Notebooks
+
+Assuming that the `ipykernel` has been installed together with the other dependencies from `requirements.txt` (step [2](#2-install-python-dependencies-inside-virtual-environment))
+
+```bash
+ipython kernel install --user --name=radcad_kernel
+```
+
+
+## First model:
+
+We refer to the section [Examples](examples.md) to see various models and references to tutorials. 
+
+Run `predator-prey-sd.py` to check if radCAD has been successfuly installed. This is a script version of the Jupyter Notebooks model `predator-prey-sd.ipynb` that is refered to in the section [Examples](/examples/#predator-prey).
+
+
+```bash
+cd examples/predator_prey_sd
+python predator-prey-sd.py
+```
+
+Did you get the expected output, as follows?
```

Comment:
New users shouldn't need to check if the results are expected. This is fine for when running tests or as an example, but probably overwhelming for a new user.